addons/binding/org.openhab.binding.rotelra1x/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,94 @@
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="source" typeId="source"/>
+            <channel id="frequency" typeId="frequency"/>
+            <channel id="dimmer" typeId="dimmer"/>
+        </channels>
+        
+        <config-description>
+            <parameter name="port" type="text">
+                <label>Serial port</label>
+                <description>Select serial port (COM1, /dev/ttyS0, ...)</description>
+                <default>COM1</default>
+                <required>true</required>
+            </parameter>
+            <!-- Max volume could have been queried using the protocol, but would have 
+                 complicated the code, and it could be nice to set 100 % at less than
+                 the receiver's max. -->
+            <parameter name="maximum-volume" type="integer">
+                <label>Maximum volume</label>
+                <description>Volume level representing 100 %.</description>
+                <default>100</default>
+                <required>true</required>
+            </parameter>
+        </config-description>
+        
+    </thing-type>
+
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Turn it on or off.</description>
+        <category>Power</category>
+    </channel-type>
+
+    <channel-type id="mute">
+        <item-type>Switch</item-type>
+        <label>Mute</label>
+        <description>Mute on/off.</description>
+        <category>Mute</category>
+    </channel-type>
+
+    <channel-type id="volume">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>
+        <description>Volume control (%)</description>
+        <category>SoundVolume</category>
+        <state pattern="%d %%"></state>
+    </channel-type>
+    
+    <channel-type id="source">
+        <item-type>String</item-type>
+        <label>Source</label>
+        <description>Select input</description>
+        <state>
+            <options>
+                <option value="cd">cd</option>
+                <option value="coax1">coax1</option>
+                <option value="coax2">coax2</option>
+                <option value="opt1">opt1</option>
+                <option value="tuner">tuner</option>
+                <option value="phono">phono</option>
+                <option value="usb">usb</option>
+                <option value="aux1">aux1</option>
+                <option value="aux2">aux2</option>
+            </options>
+        </state>
+    </channel-type>
+    
+    <channel-type id="frequency" advanced="true">
+        <item-type>Number</item-type>
+        <label>Frequency</label>
+        <description>Digital input sample rate.</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="dimmer" advanced="true">
```

Comment:
maybe brightness makes more sense as a channel name, dimmer is more the control and brightness is what you control